pandas/tests/extension/test_string.py

```diff
@@ -106,6 +106,10 @@ def test_eq_with_str(self, dtype):
         assert dtype == f"string[{dtype.storage}]"
         super().test_eq_with_str(dtype)
 
+    @pytest.mark.xfail(reason="StringDtype is a string dtype")
+    def test_is_not_string_type(self, dtype):
+        super().test_is_not_string_type(dtype)
+
```

Comment:
I don't think this is an xfail , maybe just ``pass`` and not use xfail?

Reply:
Since this tests the behavior of `assert not is_string_dtype(StringDtype)`, this should fail with the changes in the PR. Moreover, I think it's still useful to test the xfail here to align with the `extension/base.py` tests and indirectly test the is_string_dtype(StringDtype) behavior

Reply:
Would it be better if I wrote a comment in the test explaining the rational if it makes sense?

Reply:
I agree that it is useful to test for this with StringDtype.

However, my understanding of xfail is that it is something that fails and should be fixed later. In this case there is nothing to fix, so this test should pass. Do you understabd xfail differently?

Maybe instead of xfail use ``pytest.mark.skip``?

Reply:
Personally, I don't necessarily associate `xfail` with "should be fixed later" since it can be a useful signal (a change for better or worse) if a test suddenly xpasses with our `xfail_strict=True` configuration. 

So for a similar reason, I would still prefer `xfail` or `skip` here since the "change for better or worse" signal isn't captured when this test is skipped

Reply:
Ok, I'd use skip then, but it's ok for me if you want to use xfail...

Reply:
> Personally, I don't necessarily associate xfail with "should be fixed later"

im the opposite.  i regularly grep for "xfail" to find things that need attention.

What about just overriding the test

```
    def test_is_not_string_type(self, dtype):
       # the behavior differs from the base class because [...]
       assert is_string_dtype(dtype)
```

Reply:
Okay will change to not use xfail (and below) then since I have a minority opinion.